packages/grid/src/Grid.css

```diff
@@ -162,6 +167,7 @@
   margin-left: 3px;
   margin-right: 0;
 }
+
 .uitk-density-ultrahigh .uitkGrid .backwardCompat .uitkControlLabel-label {
```

Comment:
these were fixed in my PR, need to ensure that change is merged into here (since the file path changed)

Reply:
This PR is not trying to implement ultrahigh density. There is still in progress. We don't need to make this part perfect yet.